src/main/java/com/alkemy/ong/configuration/CustomAccessDeniedHandler.java

```diff
@@ -0,0 +1,30 @@
+package com.alkemy.ong.configuration;
```

Comment:
Could you please move this class to `com.alkemy.ong.security `

Reply:
Corregido